src/background/injector/content-script-injector.ts

```diff
@@ -1,37 +1,37 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { BrowserAdapter } from 'common/browser-adapters/browser-adapter';
+import { Logger } from 'common/logging/logger';
 import { PromiseFactory } from 'common/promises/promise-factory';
 import { flatten } from 'lodash';
 
 export class ContentScriptInjector {
-    public static readonly jsFiles: string[] = ['bundle/injected.bundle.js'];
+    public static readonly jsFiles: string[] = ['/bundle/injected.bundle.js'];
 
     public static readonly cssFiles: string[] = [
-        'injected/styles/default/injected.css',
-        'bundle/injected.css',
+        '/injected/styles/default/injected.css',
+        '/bundle/injected.css',
     ];
 
     public static timeoutInMilliSec = 5e4;
 
     constructor(
         private readonly browserAdapter: BrowserAdapter,
         private readonly promiseFactory: PromiseFactory,
+        private readonly logger: Logger,
     ) {}
 
-    public injectScripts(tabId: number): Promise<void> {
+    public async injectScripts(tabId: number): Promise<void> {
         // We need the JS to be injected before we can continue (ie, before we resolve the promise),
         // because the tab can't receive other messages until that's done, but it's okay for the CSS
         // to keep loading in the background after-the-fact, so it's fire-and-forget.
-        this.injectCssFilesConcurrently(tabId);
-        const inject = Promise.all([this.injectJsFilesInOrder(tabId)]).then(() =>
-            Promise.resolve(),
-        );
+        this.injectCssFilesInBackground(tabId);
+        const injectJs = this.injectJsFilesInOrder(tabId);
 
-        return this.promiseFactory.timeout(inject, ContentScriptInjector.timeoutInMilliSec);
+        await this.promiseFactory.timeout(injectJs, ContentScriptInjector.timeoutInMilliSec);
     }
 
-    private injectCssFilesConcurrently(tabId: number): void {
+    private injectCssFilesInBackground(tabId: number): void {
```

Comment:
I think the language here could be a bit better: 'InBackground' seems to imply you're injecting files into background. Maybe 'FromBackground' might make more sense? Or simply `injectCssFilesToTab` since this is in background regardless.

Reply:
Fair point! Here "In Background" was meant to imply that it is something that will run in the background in a fire-and-forget manner, didn't intend it to have a meaning related to the use of the background page. Will switch to something that doesn't use the overloaded term.

Reply:
Gotcha, I completely missed that. Makes sense now.